### ISSUE TITLE
Incorrect number of viewers listed in AntMedia console 

### ISSUE THREAD

**Issue by dmillerhotwire** [ant-media/Ant-Media-Server#5279]
### Short description
__Brief description of what happened__
Create Live stream within AntMedia console. As demonstrated by Mohit Dubuey who was sharing his screen during a demo as well, it also showed incorrect number of viewers after he created a new stream and demonstrated that he himself was watching. I believe his counted his view as 5. 
For our instance, there is a markup by at least 9 but I have seen it go up by 11 as well. When there is definitively only 1 person watching, the AntMedia Console will instead show 10 viewers or 12 viewers. 

### Environment

* Operating system and version: 
* Java version: 
* Ant Media Server version:
* Browser name and version:

### Steps to reproduce
1. login to AntMedia console
2. Create new stream
3.Tune to stream

### Expected behavior
__1 viewer should be counted as 1 viewer, and so forth__

### Actual behavior
__1 viewer is counted as 10 viewers or at times 12 viewers.__

 
Video showing this behavior was shared through email and here: https://www.dropbox.com/home/Ant%20Media 

### Logs
__Place logs on [pastebin](http://pastebin.com/) or elsewhere and put links here__

Ask your questions on [Ant Media Github Discussions](https://github.com/orgs/ant-media/discussions)

**Comment by Mohit-3196** [comment]
I made some tests for different cases and this issue is happening for the cluster environment and when playing the HLS stream via the Edge load balancer. 

1. Create a cluster with origin and edge load balancers (I created a cluster using Cloudformation)
2. Publish a stream to the origin node 
3. Play the HLS stream via the edge load balancer (both via &playOrder=hls or directly .m3u8 file from the streams directory)
4. When playing the HLS stream via the edge, for the first HLS viewer, the count shows as 2 on the dashboard and thereafter it increases linearly and there is no issue.
5. There is no issue when playing the stream from the origin itself.

Here's a video demonstration of the issue
https://github.com/ant-media/Ant-Media-Server/assets/86982446/3024d8f0-48a9-42b8-8757-62f9cd56f1ea
